packages/authenticator/amplify_authenticator/lib/src/state/authenticator_state.dart

```diff
@@ -329,6 +368,44 @@ class AuthenticatorState extends ChangeNotifier {
     _setIsBusy(false);
   }
 
+  /// Select MFA preference using the values for [selectedMfaMethod]
+  Future<void> continueSignInWithMfaSelection() async {
+    if (!_formKey.currentState!.validate() || _selectedMfaMethod == null) {
```

Comment:
Can maybe try wrapping the `AuthenticatorRadioField` column in a `FormField` to avoid the second `==`

https://api.flutter.dev/flutter/widgets/FormField-class.html